core/src/components/alert/alert.md.vars.scss

```diff
@@ -80,7 +80,7 @@ $alert-md-message-empty-padding-bottom:       $alert-md-message-empty-padding-to
 $alert-md-message-empty-padding-start:        $alert-md-message-empty-padding-end !default;
 
 /// @prop - Maximum height of the alert content
-$alert-md-content-max-height:                 240px !default;
+$alert-md-content-max-height:                 264px !default;
```

Comment:
If we are trying to show 1/2 of the 6th item's height then this should be `266px`. Half the item height is `24px`, but there are 2 `1px` borders to account for which increase the max content height from `264px` to `266px`:

| `264px` | `266px` |
| --------- | -------- |
| ![Screen Shot 2021-09-24 at 11 56 56 AM](https://user-images.githubusercontent.com/2721089/134705330-9c149932-d8b9-436c-b16e-f0ed6b26f73b.png) | ![Screen Shot 2021-09-24 at 11 56 15 AM](https://user-images.githubusercontent.com/2721089/134705356-57be2df6-503d-4fd2-822c-0c18d9239306.png) |

```suggestion
$alert-md-content-max-height:                 266px !default;
```